docs/style_guide.md

```diff
@@ -0,0 +1,75 @@
+# SageRx Style Guide
+
+## Purpose
+
+This guide will help you understand how we structure this project such as table and schema names.
+
+## Table Names
+
+These also correspond to the underlying file names that create the tables. File names must be unique and correspond to the name of the model when selected and created in the warehouse.
+
+We recommend putting as much clear information into the file name as possible, including a prefix for the layer the model exists in, important grouping information, and specific information about the entity or transformation in the model.
+
+**Marts**:
+
+- Name format: [concept]s.sql
+- Concept correctly captures the content of the table, since these are user facing this is important
+- Name should be plural
+
+**Intermediates**:
+
+- Name format: int*[entity]s*[verb]s
+- Verbs should capture the business logic or transformations conducted
+- Name should be plural
+
```

Comment:
It would be good to include a few examples for each layer of the model... eventually.

Reply:
Yes that can be for later